src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/ProfessionalUserServiceImpl.java

```diff
@@ -153,7 +153,7 @@ private ResponseEntity<Object> retrieveUserProfiles(RetrieveUserProfilesRequest
 
             responseEntity = toResponseEntity(response, clazz);
             if (responseEntity.getStatusCode().is2xxSuccessful()) {
-                responseEntity = setOrgIdInGetUserResponse(responseEntity, organisationIdentifier);
+                responseEntity = setOrgIdInGetUserResponseAndSort(responseEntity, organisationIdentifier);
             }
 

```

Comment:
as per change request, new parameter should be added to sort by provided field name in asc order, but dont see change to the endpoint:

http://rd-professional-api.service.core-compute-aat.internal/refdata/external/v1/organisations/users?returnRoles=false

The API should accept a "field name" parameter which can be passed in to allow the ascending sorting of the list of users based on the field passed in , i.e. First Name in this instance.

Reply:
@RababRabab updated JIRA after discussion, no need of new parameter now